src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/IHandler.kt

```diff
@@ -17,27 +17,37 @@
 package com.exactpro.th2.conn.dirty.tcp.core.api
 
 import com.exactpro.th2.common.grpc.MessageID
-import com.exactpro.th2.common.grpc.RawMessage
+import com.exactpro.th2.common.schema.message.impl.rabbitmq.transport.RawMessage
 import io.netty.buffer.ByteBuf
 import java.util.concurrent.CompletableFuture
 import javax.annotation.concurrent.ThreadSafe
+import com.exactpro.th2.common.grpc.RawMessage as ProtoRawMessage
 
 /**
  * Handles protocol messages and events, maintains session on a set of [channels][IChannel] belonging to a single session
  */
 @ThreadSafe
 interface IHandler : AutoCloseable {
     /**
-     * Sends provided [message]
+     * Sends provided protobuf [message]
      */
-    fun send(message: RawMessage): CompletableFuture<MessageID>
+    fun send(message: ProtoRawMessage): CompletableFuture<MessageID> {
+        throw UnsupportedOperationException("Protobuf protocol isn't supported to send message")
+    }
+
+    /**
+     * Sends provided transport [message]
+     */
+    fun send(message: RawMessage): CompletableFuture<MessageID> {
+        throw UnsupportedOperationException("th2 transport protocol isn't supported to send message")
+    }
```

Comment:
Maybe, it would be better to keep the single method but change its signature to a more generic one.
We will have to make changes in the connect anyway in order to implement support for the transport protocol. So, I don't see any reason why don't update the original method. At least in the final version when we will be close to the release

Reply:
Generic method should have the (buff: ByteBuf, properties: Map<String, String>, eventId: EventID), but it contains proto **EventID** that's meat we will migrate to new release when event transport router is ready.
Let's extend current API in minor version and make a final major release after migration to event transport router

Reply:
Okay, let's go with that